### ISSUE TITLE
Doesn't get query parameter value from datatables request?

### ISSUE THREAD

**Issue by zainul** [balderdashy/sails#3149]
Hello , i am use datatables and use server side processing, but when i will parse the paramater that given from datatables the sails doesn't show the value. i just get the last parameter in array column that shown. but when i use in expressJS 3 same as the sails i use req.query i get the complete parameter ??

what different req.query in Sails vs ExpressJS 3 ??

**Comment by tejanium** [comment]
What @zainul trying to say is:

output from `req.query` in the controller is wrong:

```javascript
// users controller
module.exports = {
	index: function (req, res) {
    return res.send(req.query);
  },
};
```

when visit `http://localhost:1337/user?foo[0][foo]=foo&foo[0][bar]=bar`

expected:
```javascript
{
  foo: [
    {
      foo: "foo",
      bar: "bar"
    }
  ]
}
```

got:
```javascript
{
  foo: [
    {
      bar: "bar"
    }
  ]
}
```

**Comment by zainul** [comment]
Thank you @tejanium for completely my question, yes my expected response like your comment.. :+1:

**Comment by tejanium** [comment]
is this expected behavior or bug?

**Comment by sailsbot** [comment]
Thanks for posting, @zainul.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
